WordPress/build.gradle

```diff
@@ -287,6 +289,29 @@ android {
         }
     }
 
+    signingConfigs {
+        if (["uploadStoreFile", "uploadStorePassword", "uploadKeyAlias", "uploadKeyPassword"].count { !secretProperties.containsKey(it) } == 0) {
+            logger.info("App signing properties found in secrets.properties, configuring signing for release builds.")
+            release {
+                storeFile = rootProject.file(secretProperties.get("uploadStoreFile"))
+                storePassword = secretProperties.get("uploadStorePassword")
+                keyAlias = secretProperties.get("uploadKeyAlias")
+                keyPassword = secretProperties.get("uploadKeyPassword")
+            }
+            android.buildTypes.release.signingConfig = android.signingConfigs.release
```

Comment:
Question (❓): Can you explain what this actually ended-up being necessary for? 🤔

Reply:
Without this line, `release` build type has no `signingConfig` applied. You can verify this by commenting this line and running `signingReport` task

Reply:
Great, thanks for the clarification on that @wzieba ! 🙇